crates/task-impls/src/quorum_proposal/mod.rs

```diff
@@ -166,7 +166,7 @@ impl<TYPES: NodeType, I: NodeImplementation<TYPES>> QuorumProposalTaskState<TYPE
                     }
                     ProposalDependency::ValidatedState => {
                         if let HotShotEvent::ValidatedStateUpdated(view_number, _) = event {
-                            *view_number + 1
+                            *view_number
```

Comment:
This seems off to me, the proposal is dependant on the state for the current view?  Shouldn't it be for the last view (i.e the parent)

Reply:
Is this due to the `view_number` parameter in this event has the same name as the argument of this function? I think the `view_number` in the event should be one view older than the `view_number` of the function argument, and here we should set the view number to be the latter.

Reply:
Fixed